pyccel/ast/builtins.py

```diff
@@ -480,6 +481,52 @@ def arg(self):
     def __str__(self):
         return f'float({self.arg})'
 
+# ===========================================================================
+class PythonRound(PyccelFunction):
+    """
+    Class representing a call to Python's native round() function.
+
+    Class representing a call to Python's native round() function
+    which rounds a float or integer to a given number of decimals.
+
+    Parameters
+    ----------
+    number : TypedAstNode
+        The number to be rounded.
+    ndigits : TypedAstNode, optional
+        The number of digits to round to.
+    """
+    __slots__ = ('_class_type',)
+    name = 'round'
+    _rank = 0
+    _shape = None
+    _order = None
+
+    def __init__(self, number, ndigits = None):
+        if ndigits is None:
+            self._class_type = PythonNativeInt()
+        else:
+            self._class_type = number.class_type
```

Comment:
I think `Booleans` should be handled separately.
```Python
if __name__ == '__main__':
    print(round(True))
```
does not work well in C.

Reply:
Ah, I didn't realise that this would execute in Python :sweat_smile:

Reply:
Fixed : [c618c2c](https://github.com/pyccel/pyccel/pull/996/commits/c618c2c3a85f858b4556bcc3b3257bfcd990b230)

And test added : f12a617